### ISSUE TITLE
Error Trying to Setup AWS Code Deploy

### ISSUE THREAD

**Issue by DavidTPate** [travis-ci/travis.rb#260]
I'm attempting to use the `travis setup codedeploy` mentioned in the [documentation](http://docs.travis-ci.com/user/deployment/codedeploy/) but I'm receiving the error shown below when executing the command. I've been able to run other `travis setup` commands successfully (specifically `elasticbeanstalk` and `s3` without issue). I've also included my Travis CLI version and Ruby version for reference.

```bash
davids-mbp:/ david.pate$ travis setup codedeploy
/usr/local/lib/ruby/gems/2.1.0/gems/travis-1.7.4/lib/travis/cli/setup.rb:40:in `const_get': cannot load such file -- travis/cli/setup/code_deploy (LoadError)
        from /usr/local/lib/ruby/gems/2.1.0/gems/travis-1.7.4/lib/travis/cli/setup.rb:40:in `service'
        from /usr/local/lib/ruby/gems/2.1.0/gems/travis-1.7.4/lib/travis/cli/setup.rb:59:in `service'
        from /usr/local/lib/ruby/gems/2.1.0/gems/travis-1.7.4/lib/travis/cli/setup.rb:54:in `run'
        from /usr/local/lib/ruby/gems/2.1.0/gems/travis-1.7.4/lib/travis/cli/command.rb:198:in `execute'
        from /usr/local/lib/ruby/gems/2.1.0/gems/travis-1.7.4/lib/travis/cli.rb:64:in `run'
        from /usr/local/lib/ruby/gems/2.1.0/gems/travis-1.7.4/bin/travis:18:in `<top (required)>'
        from /usr/local/bin/travis:23:in `load'
        from /usr/local/bin/travis:23:in `<main>'
davids-mbp:/ david.pate$ travis version
1.7.4
davids-mbp:/ david.pate$ ruby -v
ruby 2.1.4p265 (2014-10-27 revision 48166) [x86_64-darwin14.0]
```

**Comment by BanzaiMan** [comment]
There was a packaging error with 1.7.4. I've released 1.7.5, which should fix this.

Thanks for reporting.

**Comment by DavidTPate** [comment]
Grabbed the latest version of the CLI using the updated [Installation Instructions](https://github.com/travis-ci/travis.rb#installation) and the command is working as expected now.

Thanks!
